frontend/src/Table.js

```diff
@@ -106,7 +106,8 @@ const ComparisonTable = (props) => {
   const gridRef = useRef(); // Optional - for accessing Grid's API
   const [columnDefs, _setColumnDefs] = useState();
   function setColumnDefs(colNames) {
-    let defs = colNames.map(n => ({field: n}) )
+    // TODO: Are these widths ok?
+    let defs = colNames.map(n => ({field: n, minWidth: 400}) )
```

Comment:
@Sigfried I chose `400` and that seems ok as a static number for now. We can do something more complex with this later.

Reply:
As for my idea with coloring rows, we can use this: 
https://blog.ag-grid.com/conditional-formatting-for-rows-in-ag-grid/